dev/devicelab/lib/tasks/perf_tests.dart

```diff
@@ -1339,16 +1348,55 @@ class WebCompileTest {
     });
   }
 
-  /// Obtains the size and gzipped size of a file given by [fileName].
-  static Future<Map<String, int>> getSize(String fileName, {required String metric}) async {
+  /// Obtains the size and gzipped size of both [dartBundleFile] and [buildDir].
+  static Future<Map<String, int>> getSize({
+    /// Mapping of metric key name to file system path for directories to measure
+    Map<String, String> directories = const <String, String>{},
+    /// Mapping of metric key name to file system path for files to measure
+    Map<String, String> files = const <String, String>{},
+    required String metric,
+  }) async {
     final Map<String, int> sizeMetrics = <String, int>{};
 
-    final ProcessResult result = await Process.run('du', <String>['-k', fileName]);
-    sizeMetrics['${metric}_dart2js_size'] = _parseDu(result.stdout as String);
+    final Directory tempDir = Directory.systemTemp.createTempSync('perf_tests_gzips');
+    for (final MapEntry<String, String> entry in files.entries) {
+      final String key = entry.key;
+      final String filePath = entry.value;
+      ProcessResult result = await Process.run('du', <String>['-k', filePath]);
```

Comment:
How are you getting byte sizes with `du -k` instead of `du -b`? :P

Reply:
now using dart:io API to count bytes